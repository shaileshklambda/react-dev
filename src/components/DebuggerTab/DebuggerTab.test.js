import React from "react";
import renderer from "react-test-renderer";
import Debuggertab from "./index.js";

it("Debuggertab: default", () => {
  const component = renderer.create(<Debuggertab />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
