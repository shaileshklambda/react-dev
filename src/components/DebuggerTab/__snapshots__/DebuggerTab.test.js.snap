// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debuggertab: default 1`] = `
<div
  className="colAdjust col-md-12 col-xs-12"
>
  <div
    className="Debuggertab"
  >
    <div
      className="bug_list_tab activeParent"
      id="uncontrolled-tab-example"
    >
      <ul
        className="nav nav-tabs"
        role="tablist"
      />
      <div
        className="tab-content"
      />
    </div>
  </div>
</div>
`;
